.github/workflows/size-limit.yml

```diff
@@ -6,6 +6,11 @@ on:
 jobs:
   size:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
why is this one running on 12 instead of 14?

Reply:
Didn't want to mess with that one yet, was hoping it would prevent it from failing. I don't think it has.

Reply:
Once this is merged, if the next one fails, I'll look into it more.